ansible/roles_ocp_workloads/ocp4_workload_vertical_pod_autoscaler/tasks/remove_workload.yml

```diff
@@ -0,0 +1,59 @@
+---
+# Implement your Workload removal tasks here
+
+
+# New VPA operator
```

Comment:
New? :-) Also tox will probably throw up on the 2 empty lines

Reply:
Strangely, it did not.

Squeezed them anyway.